fastlane/Fastfile

```diff
@@ -320,24 +434,19 @@ platform :ios do
   # This lane finalizes the hotfix branch.
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane finalize_hotfix_release [skip_confirm:<skip confirm>]
-  #
-  # Example:
-  # bundle exec fastlane finalize_hotfix_release skip_confirm:true
+  # bundle exec fastlane finalize_hotfix_release
   #####################################################################################
   desc 'Performs the final checks and triggers a release build for the hotfix in the current branch'
-  lane :finalize_hotfix_release do |options|
-    ios_finalize_prechecks(options)
+  lane :finalize_hotfix_release do
+    # Verify that the current branch is a release branch. Notice that `ensure_git_branch` expects a RegEx parameter
+    ensure_git_branch(branch: '^release/')
 
-    if prompt_for_confirmation(
-      message: 'Ready to push changes to remote and trigger the release build?',
-      bypass: ENV.fetch('RELEASE_TOOLKIT_SKIP_PUSH_CONFIRM', nil)
-    )
-      push_to_git_remote(tags: false)
-      trigger_release_build(branch_to_build: "release/#{app_version}")
-    else
-      UI.message('Aborting hotfix finalization. See you later 👋')
-    end
+    # Verify that there's nothing in progress in the working copy
+    ensure_git_status_clean
+
+    UI.important("Triggering hotfix build for version: #{release_version_current}")
+    UI.user_error!('Aborted by user request') if !options[:skip_confirm] && !UI.confirm('Do you want to continue?')
+    trigger_release_build(branch_to_build: "release/#{release_version_current}")
```

Comment:
I don't remember where we discussed this already, but I if I recall correctly the conclusion was that the push command should have remained in the lane?

Otherwise, we might run into a scenario where the release manager has changes locally that they assume would be pushed (like the other finalization lanes do) but the CI build is triggered on the latest commit on the remote.

Or... did we conclude that there should be no local change because all changes in hotfixes happen via PRs and are already in the remote?

Reply:
That's a great point. I don't think I was a part of that discussion, but I think that scenario makes sense. _Most likely_, all the changes will be done as PRs, but might as well push any local changes just in case. 

I added the push command in https://github.com/woocommerce/woocommerce-ios/commit/8a70f61035d4d9de7c120791bbfb210e1d716f62